sources/osgShader/ShaderProgramBuilder.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
```

Comment:
I dont understand this file, it seems that we dont need it. Why we can't just update Program